Adds Juiceboxes/Cartons + Apple juice & Choc. Milk

TehZombehz opened the pull request:
:cl:
add: Small cartons can be crafted using 1 piece of cardboard. Don't be late for school.
add: Apples can now be juiced for apple juice. Chocolate milk can now be crafted using milk and cocoa.
tweak: Chocolate bar recipe has been modified to compensate for chocolate milk. The soy milk version of the chocolate bar recipe remains unchanged.
bugfix: Grapes can now be properly juiced for grape juice using a grinder.
/:cl:

![juice](https://cloud.githubusercontent.com/assets/15373249/19821249/f9946a14-9d22-11e6-9918-abcd7ebe9c17.png)

Adds craftable small cartons and juice boxes. Sprites by Nienhaus.
Volume of small cartons is 15 units.
Also adds the ability to juice apples for apple juice.
Adds chocolate milk. Changes standard chocolate bar recipe to compensate (soy milk version of the chocolate bar recipe remains the same).
Fixes being unable to juice grapes in a grinder.

TehZombehz committed:
Adds Juiceboxes/Cartons

Adds craftable small cartons and juiceboxes.

Cheridan commented:
Pfft this is perfect

Tacolizard commented:
powercreep. these should require at least three cardboard. easily available liquid holders are already too prevalent

Iamgoofball commented on code/modules/food_and_drinks/drinks/drinks.dm:
```diff
@@ -216,6 +216,32 @@
+		switch(reagents.get_master_reagent_id())
+			if("orangejuice")
+				icon_state = "juicebox"
+				name = "orange juice box	"
```
Forgot "

TehZombehz committed:
Housekeeping

Thanks, goof.

Bawhoppen commented:
Needs to be smaller, why would this tiny juicebox hold 35% volume of a bucket

Jordie0608 commented on code/modules/food_and_drinks/drinks/drinks.dm:
```diff
@@ -216,6 +216,32 @@
+	volume = 25
+
+/obj/item/weapon/reagent_containers/food/drinks/sillycup/smallcarton/on_reagent_change()
+	if (reagents.reagent_list.len > 0)
```
This can just be `if(reagents.reagent_list.len)`

nullbear commented:
I agree. Make em smaller, but reduce cost. 

Cola cans shouldn't really hold as much as they do either, but whatever. Despite cardboard being "all access" it's often rare to see someone with lots at a time.

Oh and I'm still dead. Don't bug me.

Anonmare commented:
If they can't spill their contents, have a small reagent cap and can't be used to make IEDs or anything like that then they should be good.

Nienhaus commented:
@TehZombehz what other juices you want?

TehZombehz commented:
@Nienhaus Distinct orange, apple (though the default juice box already looks like apple), and grape juice cartons.
These can spill their contents since they are just children of paper cups. Super tiny paper cups can already hold 10u by default, which is 1/5 of a bucket. I'm not against reducing the size and cost, though. How does 15u for 1 sheet sound?

TehZombehz committed:
Acting upon suggestions

Reduce cost of crafting from 2 to 1 sheet of cardboard, reduced volume
of small cartons from 25 to 15. +Housekeeping

nullbear commented:
15u for 1 sheet is great.

jughu commented:
Atleast cardboards are limited soda cans are not just rebuild the vending machine and you get a new supply! And they hold more

Nienhaus commented:
@TehZombehz https://www.dropbox.com/s/s817zedrhz9xw4t/Misc%20drinks.dmi?dl=0 Here you go.

TehZombehz committed:
New Sprites + Fixes

Adds distinct grape, orange, and chocolate milk juice box colors (from
Nienhaus). Adds chocolate milk. Changes standard chocolate recipe to
compensate for chocolate milk change (soy milk chocolate bar recipe
remains the same). Fixes being unable to juice grapes in reagent
grinders.

optimumtact commented:
Now I can spill chocolate milk down the sink
